receiver/receivercreator/receiver.go

```diff
@@ -38,11 +37,12 @@ type receiverCreator struct {
 	nextConsumer consumer.MetricsConsumerOld
 	logger       *zap.Logger
 	cfg          *Config
-	receivers    []component.Receiver
+	responder    *observerHandler
```

Comment:
Does the name `responder` reflect the purpose? It seems to be a bit of a mismatch with its type name `observerHandler`.

Reply:
Yep I renamed observerHandler at one point. Will fix.